<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(131, 131, 131);
background-color: rgb(105, 105, 117);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 18pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 184, 255);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.479904 rgba(255, 0, 0, 255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 184, 255);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.479904 rgba(255, 0, 0, 255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 184, 255);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.479904 rgba(255, 0, 0, 255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(229, 255, 131);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}
QPushButton:hover{
background-color: rgb(111, 111, 111);
font: 15pt &quot;Segoe Print&quot;;
color: rgb(0,0,0);
border-radius: 35px;}
QPushButton:pressed {
background-color: rgb(0, 255, 127);
color: rgb(0,0,0);
font: 15pt &quot;Segoe Print&quot;;
border-radius: 35px;}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>калькулятор</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
